eva-accession-import-dbsnp2/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+spring.batch.job.names=IMPORT_DBSNP_JSON_VARIANTS_JOB
+
+# dbSNP JSON source
+parameters.dbsnpSource=
```

Comment:
The rest of our pipelines use `parameters.input` to specify the primary input file.